test/bench.t/run.t

```diff
@@ -0,0 +1,284 @@
+  $ merl-an benchmark -s 2 -p bench.ml,bench1.ml --data=test-data
+
+  $ jq '.results |= map( .metrics |= map(.value |= map(0)))' test-data/bench.json
+  {
+    "name": "Merlin benchmark",
+    "results": [
+      {
+        "name": "bench.ml",
+        "metrics": [
+          {
+            "name": "case-analysis",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "complete-prefix",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "errors",
+            "value": [
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "expand-prefix",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "locate",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "occurrences",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          },
+          {
+            "name": "type-enclosing",
+            "value": [
+              0,
+              0
+            ],
+            "units": "ms",
+            "description": "",
+            "trend": null
+          }
+        ]
+      },
+      {
+        "name": "bench1.ml",
+        "metrics": [
```

Comment:
If I understand `current-bench` correctly, you're separating the benchmarks by file here by dumping the benchmarks on each file as a separate `result`.